gatsby/content/faq/who-and-how.mdx

```diff
@@ -25,8 +25,36 @@ If you would like to support the core Matrix team as a member of the community,
 
 - [Patreon](https://patreon.com/matrixdotorg)
 - [Liberapay](https://liberapay.com/matrixdotorg)
-- Bitcoin (address: 1LxowEgsquZ3UPZ68wHf8v2MDZw82dVmAE)
-- Ethereum (address: 0xA5f9a4f9E024F6D727f7afdA9257e22329A97485)
+- Bitcoin (address:
+  <div
+    style=" 
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+   -ms-word-break: break-all;
+    word-break: break-all;
+    word-break: break-word;
+   -ms-hyphens: auto;
+   -moz-hyphens: auto;
+   -webkit-hyphens: auto;
+    hyphens: auto;"
+  >
+    1LxowEgsquZ3UPZ68wHf8v2MDZw82dVmAE)
+  </div>
+- Ethereum (address:
+  <div
+    style=" 
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+   -ms-word-break: break-all;
+    word-break: break-all;
+    word-break: break-word;
+   -ms-hyphens: auto;
+   -moz-hyphens: auto;
+   -webkit-hyphens: auto;
+    hyphens: auto;"
+  >
+    0xA5f9a4f9E024F6D727f7afdA9257e22329A97485)
+  </div>
```

Comment:
I think we can just add `overflow-wrap: break-word;` to the whole [`.mxcontent`](https://github.com/matrix-org/matrix.org/blob/65e38713c5d6fe809d0f53033873ae1d6bfb90b0/gatsby/static/css/webflow-overrides.css#L300-L302) or `body` in the CSS.

It looks like it has enough browser support on its own as well, https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap#browser_compatibility

---


For reference, I see that it's used on GitHub and StackOverflow everywhere as well.